app/src/main/java/org/piwigo/ui/settings/SettingsPreferences.java

```diff
@@ -0,0 +1,25 @@
+package org.piwigo.ui.settings;
+
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+
+import org.piwigo.PiwigoApplication;
+
+public class SettingsPreferences {
```

Comment:
What is the benefit of this class? Only that we save a few characters where the preferences are used? - Setting preferences can be done inside the SettingsActivity (as the real data is in the SharedPreference I guess it is not worth the effort to create a ViewModel in that case). So only in the usage of the settings we can write

```SettingsPreferences.getSettingPreference()```
instead of
```PreferenceManager.getDefaultSharedPreferences(PiwigoApplication.getAppContext()).getXXX()```

And we don't have typing support yet, so you'd either need to add getters for all used types like int or - what I would propose - just use the PreferenceManager directly where needed.

Reply:
@ramack Probably great idea :) . Will use it.